modules/blocks.php

```diff
@@ -238,11 +238,10 @@ function jetpack_business_hours_render( $attributes, $content ) {
 		return $content;
 	}
 
-	$start_of_week     = (int) get_option( 'start_of_week', 0 );
-	$time_format       = get_option( 'time_format' );
-	$today             = current_time( 'D' );
-	$custom_class_name = isset( $attributes['className'] ) ? $attributes['className'] : '';
-	$content           = sprintf(
+	$start_of_week = (int) get_option( 'start_of_week', 0 );
+	$time_format   = get_option( 'time_format' );
+	$today         = current_time( 'D' );
+	$content       = sprintf(
 		'<dl class="jetpack-business-hours %s">',
 		! empty( $attributes['className'] ) ? esc_attr( $attributes['className'] ) : ''
```

Comment:
Here's another pattern how to do this:

https://github.com/Automattic/jetpack/blob/f7d98562aa02f001d648b1f5c009c0738090ed2e/modules/blocks.php#L119-L129

I'm indifferent the way or the other, just wanted to bring up to your attention. :-)

Reply:
👋 @jeffersonrabb had a utility to handle this and someone will get a PR ready soon. Every server rendered block seems to need this. We should open an issue upstream in the Gutenberg repo.

Reply:
https://github.com/WordPress/gutenberg/issues/13811